meilisearch-release-changelog/v1.1.0.md

```diff
@@ -0,0 +1,206 @@
+
+Some SDKs might not include all new features—consult the project repository for detailed information. Is a feature you need missing from your chosen SDK? Create an issue letting us know you need it, or, for open-source karma points, open a PR implementing it (we'll love you for that <3).
+
+<!-- The following line should NOT be put in the official release changelogs -->
+📖 Meilisearch's official documentation will be ready for the official v1.1.0 release. Meanwhile, work-in-progress documentation is available [here]().
+
+# New features and improvements 🔥
+
+## Multi-index search: allow multiple searches in a single request
+
+Perform searches in multiple indexes in a single HTTP request using the new `/multi-search` endpoint:
+
+```bash
+curl \
+  -X POST 'http://localhost:7700/multi-search' \
+  -H 'Content-Type: application/json' \
+  --data-binary '{
+    "queries": [
+      {
+        "indexUid": "products",
+        "q": "Nike",
+        "limit": 1
+      },
+      {
+        "indexUid": "brands",
+        "q": "Nike",
+        "limit": 1
+      }
+    ]
+  }'
+```
+
+When you use the multi-index search endpoint, Meilisearch returns an array of results for each queried index:
+
+```json
+{
+	"results": [
+	  {
+	    "indexUid": "products",
+	    "hits": […],
+	    "query": "Nike",
+	    "processingTimeMs": 1,
+	    "limit": 1,
+	    "offset": 0,
+	    "estimatedTotalHits": 17
+	  },
+	  {
+	    "indexUid": "brands",
+	    "hits": […],
+	    "query": "Nike",
+	    "processingTimeMs": 0,
+	    "limit": 1,
+	    "offset": 0,
+	    "estimatedTotalHits": 7
+	  }
+	]
+}
+```
+
+Done by @dureuill in #3417
+
+## Introduce `facetStats`
+
+Queries containing the `facets` parameter now include a `facetStats` object by default:
+
+```bash
+curl \
+  -X POST 'http://localhost:7700/indexes/movies/search' \
+  -H 'Content-Type: application/json' \
+  --data-binary '{
+    "facets": ["price"]
+  }'
+```
+
+`facetStats` only lists facets containing numerical values. Facets with other types of values are not included in the `facetStats` object. This object contains the minimum (`min`) and the maximum (`max`) of each numerical facet:
+
+```json
+{
+  "hits": […],
+  "facetDistribution": {
+    "price": {…}
+  },
+  "facetStats": {
+    "price": {
+      "min": 2,
+      "max": 60
+    }
+  }
+}
+```
+
+Done by @dureuill in #3423.
+
+## New geosearch filter: `_geoBoundingBox`
+
+Use the `filter` search parameter and the `_geoBoundingBox` filter expression to obtain results contained within a specific geographic area.
+
+```bash
+curl \
+  -X POST 'http://localhost:7700/indexes/restaurants/search' \
+  -H 'Content-type:application/json' \
+  --data-binary '{ "filter": "_geoBoundingBox([45.472735, 9.184019],[45.473711, 9.185613] )" }'
+```
+
+`_geoBoundingBox([lat, lng], [lat, lng])` accepts two parameters: one array specifying the top left corner, and a second array specifying the bottom right corner.
```

Comment:
```suggestion
`_geoBoundingBox([lat, lng], [lat, lng])` accepts two parameters: one array specifying the top right corner, and a second array specifying the bottom left corner.
```

Updating the change log regarding https://github.com/meilisearch/specifications/pull/223/commits/38a715c0721f7ffce57cc85d2f2f90889d2f077b